### PR TITLE
Adds new SwiftUI iOS sample app to /Examples

### PR DESCRIPTION
**Pull request by codykerns**
This PR adds the new 'MagicWeather' SwiftUI iOS sample app to the /Examples folder, alongside the current examples.

### CODE REVIEW

File: Examples/MagicWeatherSwiftUI/Shared/Sources/Views/WeatherView.swift [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    @ObservedObject var model = WeatherViewModel.shared
+    
+    var body: some View {
+        VStack {
+            /// - Sample weather details
+            
+            Text("\(model.currentData.emoji)")
+                .padding(.top, 32.0)
+                .font(.system(size: 76))
+            Text("\(model.currentData.temperature)")
```
Comment (aboedo) [inline Examples/MagicWeatherSwiftUI/Shared/Sources/Views/WeatherView.swift:26]: could we add the unit? either to the model or just here when presenting?

**Review by aboedo — APPROVED** [review]
looks awesome! left just one minor request, ready to merge after that!

File: Examples/MagicWeatherSwiftUI/Shared/Sources/Views/WeatherView.swift (hunk 1 above)
Comment (codykerns) [reply Examples/MagicWeatherSwiftUI/Shared/Sources/Views/WeatherView.swift]: totally missed this- just pushed the fix 👍

**Review by aboedo — APPROVED** [review]

**Comment by aboedo** [comment]
@codykerns feel free to merge when ready

**Pull request merged**